zfs.go

```diff
@@ -172,6 +172,44 @@ func (d *Dataset) Clone(dest string, properties map[string]string) (*Dataset, er
 	return GetDataset(dest)
 }
 
+func (d *Dataset) Umount(force bool) (*Dataset, error) {
+	if d.Type == DatasetSnapshot {
+		return nil, errors.New("cannot umount snapshots")
+	}
+	args := make([]string, 1, 3)
+	args[0] = "umount"
```

Comment:
The actual documented subcommand is `unmount`, with `umount` as syntactic sugar.